resources/test/README.md

```diff
@@ -81,3 +81,10 @@ must include a summary of the expected results as a JSON string within a
       "type": "complete"
     }
     </script>
+
+`testharness.js` is expected to function consistently in a number of
+distinct environments. In order to verify this expectation, each functional
+test may be executed under a number of distinct conditions. These conditions
+are applied using WPT's "test variants" pattern. The available variants are
+defined in the `variants.js` file; this file must be included before
+`testharness.js`.
```

Comment:
If I'm reading the code correctly, at least one variant is required to be present, right? Please document.

Reply:
That was expected, but it wasn't required. Tests that didn't specify any variants would silently report success without actually running. Good catch! I've updated the documentation, and I've also added a check to `conftest.py` so the omission will produce an error which explains the problem.